dataflow/src/ops/trigger.rs

```diff
@@ -0,0 +1,223 @@
+use std::collections::HashMap;
+
+use prelude::*;
+
+use hyper::Client;
+use tokio_core::reactor::Core;
+use serde::Serialize;
+use serde_json;
+use std::thread;
+
+/// Applies the identity operation to the view. Since the identity does nothing,
+/// it is the simplest possible operation. Primary intended as a reference
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct Trigger {
+    us: Option<IndexPair>,
+    src: IndexPair,
+    trigger: TriggerType,
+    key: Vec<usize>,
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub enum TriggerType {
+    // url to the blender, group name
+    GroupCreation{ url: String, group: String },
+}
+
+impl Trigger {
+    /// Construct a new Trigger operator.
+    pub fn new(src: NodeIndex, trigger: TriggerType, key: Vec<usize>) -> Trigger {
+        assert_eq!(key.len(), 1);
+        Trigger {
+            us: None,
+            src: src.into(),
+            trigger: trigger,
+            key: key,
+        }
+    }
+
+    fn rpc<Q: Serialize>(
+        &self,
+        path: &str,
+        requests: Vec<Q>,
+        url: &String,
+    ) where Q: Send {
+        use hyper;
+        let url = format!("{}/{}", url, path);
+        let requests: Vec<hyper::Request> = requests.iter().map(|req| {
+            let mut r = hyper::Request::new(hyper::Method::Post, url.clone().parse().unwrap());
+            r.set_body(serde_json::to_string(&req).unwrap());
+            r
+        }).collect();
+
+        thread::spawn(move || {
+            let mut core = Core::new().unwrap();
+            let client = Client::new(&core.handle());
+            for r in requests {
+                let work = client.request(r);
+                core.run(work).unwrap();
+            }
+        });
+    }
+
+    fn trigger(&self, gids: Vec<DataType>) {
```

Comment:
These are only g(roup)ids if the `TriggerType` is `GroupCreation`, right? Maybe just use `ids`?